packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -2582,15 +2593,13 @@ def _update_round(self) -> None:
         """
         background_result: Optional[Tuple[BaseSynchronizedData, Any]] = None
         if self.abci_app.is_termination_set:
-            background_result: Optional[
-                Tuple[BaseSynchronizedData, Any]
-            ] = self.abci_app.background_round.end_block()
+            background_result = self.abci_app.background_round.end_block()
         if background_result is not None and not self._termination_called:
             # when the background round returns, it takes priority over normal rounds
             # because the BackgroundRound never ends, we should only take into account
             # its response only once
             self._termination_called = True
-            result = background_result
+            result: Optional[Tuple[BaseSynchronizedData, Any]] = background_result
```

Comment:
this `Any` should be an `Enum`, representing some sort of `Event`, correct?

Reply:
Correct, again as above the definition of result was like that, and I didn't see a reason to change it.